content/zh/docs/toolbox/metering-and-billing/enable-billing.md

```diff
@@ -5,3 +5,39 @@ description: "启用 KubeSphere 中的计费功能，查看一段时间内您资
 linkTitle: "启用计费"
 weight: 15420
 ---
+
+## 开启计费功能
+
+默认计费功能是关闭的，可以通过以下步骤开启计费功能：
```

Comment:
According to https://github.com/kubesphere/ks-installer/pull/1409 and https://github.com/kubesphere/ks-installer/pull/1406 , this feature is enabled by default. So I guess you are just given an opposite description.

And I would recommend that telling users about the ClusterConfigration.

Reply:
In fact metering is enabled by default, but the other part billing was disabled by by default with invalid resource price -1. In other words, with default installation metering is enabled and billing is disabled. User can get billing info by modify metering ConfigMap.